pkg/network/tracer/tracer_windows.go

```diff
@@ -218,3 +221,25 @@ func (t *Tracer) DebugCachedConntrack(ctx context.Context) (interface{}, error)
 func (t *Tracer) DebugHostConntrack(ctx context.Context) (interface{}, error) {
 	return nil, ebpf.ErrNotImplemented
 }
+
+func newHttpMonitor(c *config.Config, dh driver.Handle) http.Monitor {
+	if !c.EnableHTTPMonitoring {
+		return nil
+	}
```

Comment:
As mentioned above, currently having `EnableHTTPHTTPSMonitoringViaETW` enabled doesn't do anything unless http monitoring is also explicitly enabled - I think enabling etw monitoring shouldn't require 2 configuration values